### PR TITLE
fix(api): Allow to get every service through getServiceHandler

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#13214.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

File: contrib/grpcplugins/action/clair/main.go [hunk 1]
```diff
@@ -38,10 +38,11 @@ func (actPlugin *clairActionPlugin) Manifest(ctx context.Context, _ *empty.Empty
 func (actPlugin *clairActionPlugin) Run(ctx context.Context, q *actionplugin.ActionQuery) (*actionplugin.ActionResult, error) {
 	// get clair configuration
 	fmt.Printf("Retrieve clair configuration...")
-	serv, err := grpcplugins.GetExternalServices(actPlugin.HTTPPort, "clair")
+	servs, err := grpcplugins.GetServices(actPlugin.HTTPPort, "clair")
 	if err != nil {
 		return actionplugin.Fail("Unable to get clair configuration: %s", err)
 	}
+	serv := servs[0]
```
Comment (richardlt) [inline contrib/grpcplugins/action/clair/main.go:45]: Check length before

File: engine/api/api_routes.go [hunk 2]
```diff
@@ -423,7 +423,7 @@ func (api *API) InitRouter() {
 	// Engine µServices
 	r.Handle("/services/register", Scope(sdk.AuthConsumerScopeService), r.POST(api.postServiceRegisterHandler, MaintenanceAware()))
 	r.Handle("/services/heartbeat", Scope(sdk.AuthConsumerScopeService), r.POST(api.postServiceHearbeatHandler))
-	r.Handle("/services/{type}", Scope(sdk.AuthConsumerScopeService), r.GET(api.getExternalServiceHandler))
```
Comment (richardlt) [inline engine/api/api_routes.go:426]: /services?type=clair should be better no ?

File: engine/api/services/external.go [hunk 3]
```diff
@@ -11,8 +12,26 @@ import (
 	"github.com/ovh/cds/sdk/log"
 )
 
+// ExternalService represents an external service
+type ExternalService struct {
+	sdk.Service `json:"-"`
```
Comment (richardlt) [inline engine/api/services/external.go:17]: We realy need this?

File: engine/api/services/external.go [hunk 4]
```diff
@@ -11,8 +12,26 @@ import (
 	"github.com/ovh/cds/sdk/log"
 )
 
+// ExternalService represents an external service
+type ExternalService struct {
```
Comment (richardlt) [inline engine/api/services/external.go:16]: I think this type can be removed, ServiceConfiguration will be enough.

**Pull request merged**